### PR TITLE
fix:removed try and catch from the app.ts

### PR DESCRIPTION
**Pull request by vishwab1**
Removed the try and catch block from app.ts

### CODE REVIEW

File: src/models/skillMaster.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { DataTypes } from 'sequelize';
```
Comment (Hari-1027) [inline src/models/skillMaster.ts:1]: no need skills master

File: src/models/classMaster.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+export const classMaster = AppDataSource.define(
+  'class_master',
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    name: {
```
Comment (Hari-1027) [inline src/models/classMaster.ts:12]: name is multi lingual

File: src/models/classMaster.ts [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    prerequisites: {
+      type: DataTypes.STRING,
```
Comment (Hari-1027) [inline src/models/classMaster.ts:17]: this is a array

File: src/models/roleMaster.ts [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    },
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    description: {
+      type: DataTypes.STRING,
+      allowNull: true,
+    },
+    sequence_num: {
```
Comment (Hari-1027) [inline src/models/roleMaster.ts:21]: sequence is not needed

File: src/models/subSkillMaster.ts [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+export const subSkillMaster = AppDataSource.define(
+  'sub_skill_master',
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    name: {
+      type: DataTypes.STRING,
```
Comment (Hari-1027) [inline src/models/subSkillMaster.ts:14]: multilingual

File: src/models/subSkillMaster.ts [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+      allowNull: false,
+    },
+    description: {
+      type: DataTypes.STRING,
+      allowNull: true,
+    },
+    sequence_num: {
+      type: DataTypes.INTEGER,
```
Comment (Hari-1027) [inline src/models/subSkillMaster.ts:22]: no sequence is needed

**Comment by vishwab1** [comment]
@HarishGangula   please review the master api

**Comment by vishwab1** [comment]
[createTable.txt](https://github.com/user-attachments/files/16979582/createTable.txt)

File: src/models/boardMaster.ts [hunk 7]
```diff
@@ -9,21 +9,25 @@ export const boardMaster = AppDataSource.define(
...
       allowNull: true,
     },
     status: {
       type: DataTypes.ENUM('draft', 'live'),
       allowNull: false,
     },
-    class_id: {
-      type: DataTypes.ARRAY(DataTypes.INTEGER),
-      allowNull: true,
+    class_names: {
```
Comment (HarishGangula) [inline src/models/boardMaster.ts:28]: This needs to be removed

**Review by HarishGangula — CHANGES_REQUESTED** [review]

File: src/models/boardMaster.ts (hunk 7 above)
Comment (vishwab1) [reply src/models/boardMaster.ts]: @HarishGangula changes done, please verify

**Review by HarishGangula — APPROVED** [review]

**Pull request merged**